README.md

```diff
@@ -1,50 +1,16 @@
-# Union Client Project
-
-## Table of Contents
-
-1. [Quickstart](#quickstart)
-2. [File Structure](#file-structure)
-3. [Testing](#testing)
-4. [Code Quality](#code-quality)
-5. [Pages](#pages)
-   - [Home](#home)
-   - [Events](#events)
-   - [Calendar](#calendar)
-   - [About](#about)
-   - [Account](#account)
-   - [Splash](#splash)
-   - [Login](#login)
-   - [Register](#register)
-   - [Error](#error)
-   - [Admin](#admin)
-6. [Components](#components)
-   - [Button](#button)
-   - [EventCard](#eventcard)
-   - [FederatedSignIn](#fedsignin)
-   - [Form](#form)
-   - [Gallery](#gallery)
-   - [Header](#header)
-   - [InputBox](#inputbox)
-   - [Layout](#layout)
-   - [LoadingSpinner](#loadingspinner)
-   - [Nav](#nav)
-   - [SearchBar](#searchbar)
-   - [Events Booking Popup Window](#events-booking-popup-window)
-7. [Dev Tools](#dev-tools)
-
-   - [Mock Data](#mock-data)
-   - [SnapShot FireBase](#snapshot-firebase)
-   - [FirebaseSnapshots](#firebasesnapshots)
-   - [Firebase Auth persistence](#firebase-auth-persistence)
-   - [How to Create New Users](#how-to-create-new-users)
-   - [Data Structure](#data-structure)
-   - [User Events](#user-events)
-   - [How To See Future and Past Events](#how-to-see-future-and-past-events)
-   - [How to see Event Details](#how-to-see-event-details)
-   - [Queries](#queries)
-   - [Event Creation](#event-creation)
-
-8. [About Us](#about-us)
+# Dot to Dot - Proof of Concept
+
+## Contributors
+Chu Lam - https://github.com/Koji47</br> 
```

Comment:
With markdown write links using `[]()`
e.g `[Chu Lam](https://github.com/Koji47)` will give you [Chu Lam](https://github.com/Koji47)